dsl-reference.md

```diff
@@ -1389,6 +1391,28 @@ Represents the configuration of an event consumption strategy.
 | any | [`eventFilter[]`](#event-filter) | `no` | Configures the workflow to wait for any of the defined events before resuming execution.<br>*Required if `all` and `one` have not been set.* |
 | one | [`eventFilter`](#event-filter) | `no` | Configures the workflow to wait for the defined event before resuming execution.<br>*Required if `all` and `any` have not been set.* |
 
+### Event Properties
```

Comment:
While the description is good, afaik it applies to an event, instead of describing the properties object, which might be confusing, no?

Reply:
We are describing the properties of an event, therefore giving the context of what the event is and then having the table with the properties seemed like a good approach. What alternative do you suggest ?